components/upload/Dragger.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Git diff is crazy.

The most important thing is just make Dragger as a ES6 style component.